scripts/valid_payload.json

```diff
@@ -5,27 +5,19 @@
     "family_of_business": "Test Business Family",
```

Comment:
since this is kind of used as an example, maybe we should start breaking these out into separate files so we have some sort of reference library of core transaction schema examples?